css/layout.css

```diff
@@ -143,4 +144,93 @@ fieldset {
 
 .changing-text {
     grid-area: c;
+}
+
+.shop {
+    grid-area: shop;
+}
+
+.search-login {
+    grid-area: l;
+}
+
+.menu-list {
+    grid-area: ml;
+}
+
+.about {
+    grid-area: ab;
+}
+
+.wheelbarrow {
+    grid-area: wh;
+}
+
+.leaf {
+    grid-area: le;
+}
+
+.woman {
+    grid-area: wo;
+}
+
+.offer {
+    grid-area: of;
+}
+
+@media screen and (min-width: 980px) {
+    .white-background {
+        grid-template-columns: repeat(11, 1fr) auto;
+        grid-template-areas: "s s s s s s s s l l l b";
+    }
```

Comment:
This isn't incorrect, but do you need to create a new grid here? Could you have set these template areas on the `.header` grid directly?

Reply:
I believed I tried it that way and had problems aligning everything and also giving the top half of the header a white background but I will try it again until I can figure it out.